src/Tickets/Seating/Libraries/Uplink.php

```diff
@@ -59,25 +64,19 @@ public function do_register(): void {
 	 * @return void
 	 */
 	public function unregister(): void {
-		$this->remove_actions();
-	}
+		remove_action( 'init', [ $this, 'register_plugin' ] );
 
-	/**
-	 * Add the action hooks.
-	 *
-	 * @since TBD
-	 */
-	public function add_actions(): void {
-		add_action( 'init', [ $this, 'register_plugin' ] );
+		$prefix = tribe( Libraries_Provider::class )->get_hook_prefix();
+		remove_action( 'stellarwp/uplink/' . $prefix . '/' . $this->et_slr_plugin_slug . '/connected', [ $this, 'store_successful_connection' ] );
 	}
 
 	/**
-	 * Remove the action hooks.
+	 * Stores a successful connection through uplink.
 	 *
 	 * @since TBD
 	 */
-	public function remove_actions(): void {
-		remove_action( 'init', [ $this, 'register_plugin' ] );
+	public function store_successful_connection(): void {
+		tribe_update_option( 'tec_tickets_seating_connected_on', time() );
```

Comment:
Since the option is being stored inside an already namespace TEC array of options, I would drop the `tec_` prefix and call the option `tickets_seating_connected_on`.

Then: does the value need to be a timestamp? It might, but if it needs to be a timestamp, then it should be verified when read ("is this a valid timestamp?"). If it's sufficient for the value to be there or not, then it's fine to store a boolean-ish value.